.github/workflows/ci.yml

```diff
@@ -175,7 +175,7 @@ jobs:
 
       - run: make flutter.build platform=${{ matrix.platform }}
                   dart-env='SOCAPP_FCM_VAPID_KEY=${{ secrets.FCM_VAPID_KEY }}
-                            SOCAPP_SENTRY_DSN=${{ secrets.SENTRY_DSN }}'
+                            SOCAPP_SENTRY_DSN=${{ startsWith(github.ref, 'refs/tags/v') && secrets.SENTRY_DSN || '' }}'
```

Comment:
Since `sentry.upload` is currently invoked only for release tags, it makes sense to set DSN for releases only as well. Yet it should in the future use separate DSN for staging builds, and upload the symbols to it.